src/lexer/Lexer.spec.ts

```diff
@@ -792,6 +792,12 @@ describe('lexer', () => {
     });
 
     describe('long integer literals', () => {
+        it('respects \'&\' suffix', () => {
```

Comment:
Not directly impacted by this code, but the test for this was missing so I added it.